src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -1353,7 +1353,19 @@ void GameObject::Use(Unit* user)
                         break;
                 }
 
-                player->KillCreditGO(info->entry, GetGUID());
+                if (Group* group = player->GetGroup())
+                {
+                    GroupReference const* ref = group->GetFirstMember();
+                    while (ref)
```

Comment:
Change to a for loop as in https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/Entities/Totem/Totem.cpp#L136-L141